packages/sdk-themes/src/themes/monite.ts

```diff
@@ -182,22 +659,221 @@ export const defaultMoniteComponents: Components<Omit<Theme, 'components'>> = {
+      },
+      {
+        props: { status: 'recurring' },
+        style: statusColors.green,
+      },
+      {
+        props: { status: 'partially_paid' },
+        style: statusColors.violet,
+      },
+      {
+        props: { status: 'paid' },
+        style: statusColors.green,
+      },
+      {
+        props: { status: 'overdue' },
+        style: statusColors.orange,
+      },
+      {
+        props: { status: 'uncollectible' },
+        style: statusColors.red,
+      },
+      {
+        props: { status: 'canceled' },
+        style: statusColors.orange,
+      },
+      {
+        props: { status: 'deleted' },
+        style: statusColors.black,
+      },
+    ],
+  },
+  MonitePayableStatusChip: {
+    defaultProps: {
+      icon: false,
+    },
+    styleOverrides: {
+      root: {
+        fontSize: '13px',
+      },
+    },
+    variants: [
+      {
+        props: { status: 'draft' },
+        style: statusColors.black,
+      },
+      {
+        props: { status: 'new' },
+        style: statusColors.blue,
+      },
+      {
+        props: { status: 'approve_in_progress' },
+        style: statusColors.orange,
+      },
+      {
+        props: { status: 'waiting_to_be_paid' },
+        style: statusColors.orange,
+      },
+      {
+        props: { status: 'partially_paid' },
+        style: statusColors.violet,
+      },
+      {
+        props: { status: 'paid' },
+        style: statusColors.green,
+      },
+      {
+        props: { status: 'canceled' },
+        style: statusColors.orange,
+      },
+      {
+        props: { status: 'rejected' },
+        style: statusColors.red,
+      },
+    ],
+  },
 };
 
+// Copied from mui/utils sourcecode since we cannot import external libraries in this file
+function isPlainObject(item: unknown) {
+  if (typeof item !== 'object' || item === null) {
+    return false;
+  }
+  const prototype = Object.getPrototypeOf(item);
+  return (
+    (prototype === null ||
+      prototype === Object.prototype ||
+      Object.getPrototypeOf(prototype) === null) &&
+    !(Symbol.toStringTag in item) &&
+    !(Symbol.iterator in item)
+  );
+}
+
+// Replaces color constant like 'divider', 'primary.main', 'neutral.80' with actual color value
+function renderColor(strVal: string, palette: PaletteOptions): string {
+  if (strVal.includes('.')) {
+    let temporaryValue: { [key: string]: any } = palette;
+    strVal.split('.').forEach((key) => {
+      temporaryValue = temporaryValue && temporaryValue[key];
+    });
+    // noinspection SuspiciousTypeOfGuard
+    if (temporaryValue && typeof temporaryValue === 'string')
+      return temporaryValue;
+  } else if (typeof palette[strVal as keyof PaletteOptions] === 'string')
+    return palette[strVal as keyof PaletteOptions] as string;
+  return strVal;
+}
+
+// Replaces color constants like 'divider', 'primary.main', 'neutral.80' with actual color values
+function renderColors<T extends { [key: string]: any }>(
```

Comment:
Am I correct in understanding that the MUI does not support `primary.main` values in the theme?

Reply:
@radist2s Yep. It leaves string values as-is in MUI v7.